core/src/main/web/outputdisplay/bko-latex.js

```diff
@@ -24,7 +24,11 @@
     return {
       link: function(scope, element, attrs) {
         scope.$watch('model.getCellModel()', function(newValue) {
-          katex.render(newValue, element[0]);
+          try {
```

Comment:
this should trigger only when the value changes anyway and the block contains only an invocation to another function which is optimized, so the performance hit should not be so bad